client/app/pages/alert/index.js

```diff
@@ -70,6 +76,22 @@ function AlertCtrl($routeParams, $location, $sce, toastr, currentUser, Query, Ev
     );
   };
 
+  this.preview = () => {
+    const notifyError = () => toastr.error('Unable to render description. please confirm your template.', { timeOut: 10000 });
+    try {
+      const result = this.alertTemplate.render(this.alert.options.template, this.queryResult.query_result.data);
+      this.alert.preview = $sce.trustAsHtml(result.escaped);
+      this.alert.previewHTML = $sce.trustAsHtml(result.raw);
```

Comment:
This needs to be sanitized before trusting.

Reply:
thank you for pointing mistake!